packages/ember-views/lib/compat/attrs-proxy.js

```diff
@@ -56,46 +37,34 @@ let AttrsProxyMixin = {
     val.update(value);
   },
 
-  willWatchProperty(key) {
-    if (this._isAngleBracket || key === 'attrs') { return; }
-
-    let attrsKey = `attrs.${key}`;
-    _addBeforeObserver(this, attrsKey, null, attrsWillChange);
-    addObserver(this, attrsKey, null, attrsDidChange);
+  _propagateAttrsToThis() {
+    let attrs = this.attrs;
+    let values = {};
+    for (let prop in attrs) {
+      if (prop !== 'attrs') {
+        values[prop] = this.getAttr(prop);
+      }
+    }
+    this.setProperties(values);
```

Comment:
although strange, set in a loop is typically faster, since the batching of setProperties comes at a cost.

Worth testing (if someone has time)